server/config.json

```diff
@@ -19,8 +19,8 @@
       "limit": "100kb"
     },
     "cors": false,
-    "errorHandler": {
-      "disableStackTrace": false
+    "strong-error-handler": {
+      "isDebugMode": false
```

Comment:
I believe the `errorhandler` section of `remoting` config was not changed, pelase revert.

@davidcheung could you please confirm?